pixart.js

```diff
@@ -1 +1,37 @@
+// commit 3 function
+function clickSquare(){
+      $(this).css("background", "green");
+    }
 
+function chooseMyC(){
+  var matchC=$('#color-field').val();
```

Comment:
Is there a way you can set this to the swatch so that you can start drawing right when you open the app?